util/ranger/ranger.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/pingcap/tidb/sessionctx/stmtctx"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/codec"
+	"unicode/utf8"
```

Comment:
Move this to the upper group.

Reply:
@shenli There seems to be a bug in git pull request because some of my commits are not pushed into this pr as it should. I have opend another PR to fix this.